conf/ref_databases.config

```diff
@@ -239,6 +247,22 @@ params {
             dbversion = "UNITE-alleuk v8.2 (https://doi.org/10.15156/BIO/786370)"
             shfile = [ "https://scilifelab.figshare.com/ndownloader/files/34994569", "https://scilifelab.figshare.com/ndownloader/files/34994572"]
         }
+        'zehr-nifh' {
+            title = "Zehr lab nifH database - version 2.5.0"
+            file = [ "https://raw.githubusercontent.com/moyn413/nifHdada2/master/nifH_dada2_v2.0.5.fasta", "https://raw.githubusercontent.com/moyn413/nifHdada2/master/nifH_dada2_phylum_v2.0.5.csv" ]
+            citation = "M. A. Moynihan & C. Furbo Reeder 2023. nifHdada2 GitHub repository, v2.0.5. Zenodo. http://doi.org/10.5281/zenodo.7996213"
```

Comment:
Just a thought: might it be better to get it from https://zenodo.org/records/7996213 instead of github? But that would require a change of the parsing script.

Reply:
You mean the fasta and csv? If it requires a change somewhere, I'd say no. If this becomes something that's frequently used, we could perhaps consider changing.